docs/docs/dev-manuals/services/discount.mdx

```diff
@@ -0,0 +1,225 @@
+import CustomImage from "@site/src/components/CustomImage";
+
+# Discount Service
+
+The discount service provides the bounded context `Discount`.
+It is responsible for managing discounts, coupons, and their usages by userso.
```

Comment:
Typo in "userso"?!

Reply:
fixed with 5cb6c4b68ae5261f254cdc5392f3bf7c0a1560e7